vehicle/mkvii/software/dashboard/dashboard.yml

```diff
@@ -0,0 +1,48 @@
+name: dashboard
+subscribe:
+- name: brakelight
+  mob: 1
+- name: bms_core
+  mob: 2 
+- name: air_control_critical
+  mob: 3 
+- name: throttle
+  mob: 4 
+
+publish:
+  - name: dashboard
+    id: 0xf
+    freq_hz: 100
+    signals:
+      - name: ready_to_drive
+        slice: 0 + 1
+        unit: 
+          type: bool
+          values:
+            t: READY TO DRIVE
+            f: NOT READY TO DRIVE
+  
+      - name: start_button_state
+        slice: 1 + 1
+        unit: 
+          type: bool
+          values:
+            t: PRESSED
+            f: NOT PRESSED
+
+      - name: driver_estop_ss
```

Comment:
nit: CANViewer expects shutdown sense signals to follow the pattern `ss_<node>`, could we change this and the BOTS to match that convention?